etc/config-template.toml

```diff
@@ -81,6 +81,11 @@
 ## The number of max concurrent streams/requests on a client connection.
 # grpc-concurrent-stream = 1024
 
+## Limit the memory size can be used by gRPC. Default is unlimited.
+## gRPC usually works well to reclaim memory by itself. Limit the memory in case OOM
+## is observed. Note that limit the usage can lead to potential stall.
+# grpc-memory-pool-quota = "32G"
```

Comment:
em, is default 32G too large?

Reply:
32G just indicates "big quota size". Because exceeding the limit may cause some unpredictable stall (meaning that the stream is disconnected, the channel refuses to connect, etc.). By default, I don't want this to cause potential problems.